README.md

```diff
@@ -16,29 +34,254 @@ Karpenter improves the efficiency and cost of running workloads on Kubernetes cl
 * **Removing** the nodes when they are no longer needed
 * **Consolidating** existing nodes onto cheaper nodes with higher utilization per node
 
-## Production Readiness Status
-- API-Version: AKS Karpenter Provider is currently in alpha (`v1alpha2`).
-- AKS-Feature State: [Node Auto Provisioning is currently in preview](https://learn.microsoft.com/en-gb/azure/aks/node-autoprovision?tabs=azure-cli)
 
-## Installation: Managed Karpenter (AKA Node Auto Provisioning) 
-The Node Auto Provisioning Preview, runs Karpenter as a managed addon similar to Managed Cluster Autoscaler. 
+## Node Auto Provisioning (NAP) vs. Self-hosted
+
+Karpenter provider for AKS can be used in two modes:
+* **[Node Auto Provisioning (NAP)](https://learn.microsoft.com/en-gb/azure/aks/node-autoprovision?tabs=azure-cli) mode** (preview): Karpenter is run by AKS as a managed addon similar to managed Cluster Autoscaler. This is the recommended mode for most users. Follow the instructions in Node Auto Provisioning documentation to use Karpenter in that mode.
+* **Self-hosted mode**: Karpenter is run as a standalone deployment in the cluster. This mode is useful for advanced users who want to customize or experiment with Karpenter's deployment. The rest of this page describes how to use Karpenter in self-hosted mode.
+
+## Known limitations
+
+* Only AKS clusters with Azure CNI Overlay + Cilium networking are supported.
+* Only Linux nodes are supported.
+
+## Installation (self-hosted)
+
+This guide shows how to get started with Karpenter by creating an AKS cluster and installing Karpenter.
+
+### Install utilities
+
+Install these tools before proceeding:
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [Helm](https://helm.sh/docs/intro/install/)
+* [jq](https://stedolan.github.io/jq/) - used by some of the scripts below
+* [yq](https://mikefarah.gitbook.io/yq/) - used by some of the scripts below
+
+### Create a cluster
+
+Create a new AKS cluster with the required configuration, and ready to run Karpenter using worload identity.
+
+> Note: You can use `hack/deploy/create-cluster.sh <cluster-name> <resource-group>` to automate the following steps. 
+
+Set environment variables:
+
+```bash
```

Comment:
export subscription?

Reply:
I don't think it is used anywhere explicitely